<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgSettings</class>
 <widget class="QDialog" name="dlgSettings">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>366</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Game Settings</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="btnBoxOptions">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>450</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblApplication">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>10</y>
     <width>120</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Application</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cboApplication">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>5</y>
     <width>300</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblDebug">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>47</y>
     <width>120</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>WINEDEBUG</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDebug">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>43</y>
     <width>300</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblPrefix">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>85</y>
     <width>120</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>WINEPREFIX</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPrefix">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>80</y>
     <width>300</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cboBottle">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>79</y>
     <width>300</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtGameDir">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>117</y>
     <width>260</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="btnGameDir">
   <property name="geometry">
    <rect>
     <x>419</x>
     <y>117</y>
     <width>36</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblGameDir">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>121</y>
     <width>150</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Game Directory</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblGraphics">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>158</y>
     <width>150</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Hi-Res Graphics</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cboGraphics">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>153</y>
     <width>300</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkAdvanced">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>225</y>
     <width>280</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Advanced Options</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtProgram">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>258</y>
     <width>300</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblProgram">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>262</y>
     <width>140</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Wine Program</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPatchClient">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>294</y>
     <width>300</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblPatchClient">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>298</y>
     <width>150</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Patch Client DLL</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkx86">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>64-bit Client</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnBoxOptions</sender>
   <signal>accepted()</signal>
   <receiver>dlgSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBoxOptions</sender>
   <signal>rejected()</signal>
   <receiver>dlgSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
